src/cbmc/fault_localization.cpp

```diff
@@ -192,6 +194,37 @@ void fault_localizationt::report(irep_idt goal_id)
                    << eom;
 }
 
+void fault_localizationt::report_xml(irep_idt goal_id, xmlt& xml)
+{
+  xml.new_element("method").data="linear fault localization";
+
+  if(goal_id==ID_nil)
+    goal_id=failed->source.pc->source_location.get_property_id();
+
+  xml.set_attribute("property", id2string(goal_id));
+
+  lpointst &lpoints = lpoints_map[goal_id];
+
+  if(lpoints.empty())
+  {
+    xml.new_element("result").data="unable to localize fault";
+    return;
+  }
+
+  debug() << "Fault localization scores:" << eom;
```

Comment:
It's neither an error nor debug information. It's less important status information that some people might be interested in, but otherwise would clutter the output.